README.md

```diff
@@ -2,28 +2,29 @@
+Deploy the operator and chat with Hermes in 2 or 3 steps. Note that you need a NVIDIA GPU with 8GB of RAM.
 
-### Why Use Prem Operator?
+1. Install the NVIDIA Operator if not present already
 
-- 🚀**Efficiency in Deployment:** Prem Operator abstracts and simplifies the deployment process of AI models on Kubernetes, making it quicker and more efficient. Users can deploy models across different engines without diving into the specifics of each, saving time and resources.
-- 📐**Flexibility and Scalability:** Catering to the needs of modern AI applications, it offers unmatched flexibility, allowing for the deployment of models across various engines and configurations. Its scalable nature ensures that as your AI needs grow, Prem Operator grows with you, adapting to new requirements and computational demands.
-- 🏗**Unified Management:** By providing a unified framework for AI model deployment, Prem Operator eliminates the fragmentation typically associated with managing multiple AI engines and their deployments. It brings consistency to your operations, enabling easier management and monitoring of AI models.
-- ⚡**Enhanced Performance:** Leveraging the specific strengths of supported AI engines, Prem Operator ensures that AI models are deployed in environments where they perform best, optimizing computational resources and enhancing overall model performance.
-- 🤝**Innovation and Collaboration:** As an open-source project, Prem Operator encourages innovation and collaboration within the community. It enables developers to contribute to a growing ecosystem, enhancing the tool's capabilities and ensuring it remains at the forefront of AI model deployment technology.
+2. Install the Prem Operator
+    ```
+    $ helm install latest oci://registry-1.docker.io/premai/prem-operator-chart
+    ```
 
-For developers and contributors interested in diving deeper into Prem Operator, we've made available a comprehensive [**Developer Guide**](./docs/developer_guide.md), a detailed [**Contribution Guide**](./docs/deployment.md), and a step-by-step [**Deployment Guide**](./docs/deployment.md). These resources are designed to help you get started, contribute to the project, and deploy AI models with ease.
+3. Deploy the Hermes + Big-AGI example and forward the ports
+    ```
+    $ kubectl -f exmaples/big-agi.yaml
+    $ kubectl port --address localhost --port 8080 forward big-agi-service --port 3000
+    ```
 
-In summary, Prem Operator is an essential tool for developers and organizations looking to deploy AI models efficiently, flexibly, and scalably within Kubernetes environments. Its design philosophy prioritizes ease of use, performance optimization, and collaborative improvement, making it an ideal choice for cutting-edge AI deployments.
+If you browse to localhost:3000 you'll be able to beging chatting as soon as
+LocalAI downloads the model.
```

Comment:
```suggestion
LocalAI downloads the model. All queries after the first one, where the model gets downloaded should be much faster.
```

Reply:
This is clear in the video (which I haven't converted to a GIF yet, but you can see here https://youtu.be/MN8WpMG9PKs)

I think the information here has to be really minimal.